packages/server/test/integration/conduits.test.ts

```diff
@@ -41,6 +41,7 @@ describe('Conduits', () => {
     expect(conduit.config).toEqual(state.channelConfig)
 
     state.conduit = conduit
+    await app.instances.lifetimes.stop(conduit.id)
```

Comment:
Why stop it?

Reply:
Messes up test metrics because it sends some events